Packs/Core/ReleaseNotes/1_3_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Investigation & Response
+ - Added support for the *username* argument in the ***xdr-get-endpoints*** command.
```

Comment:
It's a bit confusing that we have this RN but we don't change this integration at all... do we have a way to hide it? if not, let's rephrase it somehow so customers won't get confused.